.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    paths:
+      - "**.md"
+    branches:
+      - "main"
+  pull_request:
+    paths:
+      - "**.md"
+    branches:
+      - "main"
+
+jobs:
+  mdlint:
+    name: Lint Markdown
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # 2.3.4
+
+      - name: Run markdownlint
+        uses: nosborn/github-action-markdown-cli@9fc95163471d6460c35cccad13645912aaa25d5f # 1.1.1
+        with:
+          files: "."
+          config_file: ".github/markdownlint.json"
+          ignore_files: "analyzer-comments/*"
```

Comment:
I have no idea why this doesn't work: https://github.com/marketplace/actions/markdownlint-cli

I tried
```
"analyzer-comments/*"
"analyzer-comments/**/*"
"analyzer-comments/**"
```

Reply:
Tried "analyzer-comments" or "analyzer-comments/" ?
Looking at https://github.com/igorshubovych/markdownlint-cli, it seems like the globbing is used to reference specific files. But the CLI parameter also allows a directory to be ignored (allows file, directory or glob). The README for the link you have given mentions the same.